packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -2679,6 +2679,79 @@ it('should handle resize with touch', () => {
   expect(headerCells[2].style.width).toBe('100px');
 });
 
+it.each(['px', '%', 'em', 'rem', 'vw', 'vh', 'ch', 'ex'])(
+  'should handle resize when widths are string units. (E.g. %s)',
+  (unit) => {
+    jest
+      .spyOn(HTMLElement.prototype, 'getBoundingClientRect')
+      .mockReturnValue({ width: 100 } as DOMRect);
+
+    const columns: Column<TestDataType>[] = [
+      {
+        id: 'name',
+        Header: 'Name',
+        accessor: 'name',
+        width: `20${unit}`,
+      },
+      {
+        id: 'description',
+        Header: 'description',
+        accessor: 'description',
+        width: `20${unit}`,
+      },
+      {
+        id: 'view',
+        Header: 'view',
+        Cell: () => <>View</>,
+        width: `60${unit}`,
+      },
+    ];
+
+    const { container } = renderComponent({
+      columns,
+      isResizable: true,
+    });
+
+    const rows = container.querySelectorAll('.iui-table-body .iui-table-row');
+    expect(rows.length).toBe(3);
+
+    const resizers = container.querySelectorAll(
+      '.iui-table-resizer',
+    ) as NodeListOf<HTMLDivElement>;
+
+    expect(resizers).toBeTruthy();
+    resizers.forEach((resizer) => expect(resizer).toBeTruthy());
```

Comment:
nit: these can be removed
```suggestion
```

Reply:
Since I am testing resizing using the first and the second resizer, `resizers` is needed.

I had initially kept L2715-2716 since they were there in many of the tests. But removed it since it's not needed. (https://github.com/iTwin/iTwinUI/pull/1799/commits/0e6f208a9ae7cfc7b4105353e79e8930282ec7b7)

Reply:
Ah yeah. I think the `toBeTruthy()` checks for `resizers` could still be removed, but it's ok